www/front_src/src/Resources/Details/tabs/Timeline/index.tsx

```diff
@@ -95,42 +111,49 @@ const TimelineTab = ({ links }: Props): JSX.Element => {
     return sendRequest({
       endpoint: timelineEndpoint,
       parameters: {
-        page,
+        page: toPage,
         limit,
         search: getSearch(),
       },
     }).then((retrievedListing) => {
       const { meta } = retrievedListing;
       setTotal(meta.total);
+      setLoadingMoreEvents(false);
```

Comment:
I think it's better to put it in a `finally` to reset it even if there is a request error